metadetect/lsst_metadetect.py

```diff
@@ -84,21 +86,25 @@ def _measure(self, mbobs, shear_str):
 
         res = self._fitter.go(mbobs_list)
         if res is not None:
-            res = self._add_positions_and_psf(medsifier, res, shear_str)
+            res = self._add_positions_and_psf(medsifier, res, shear_str, mbobs_list)
 
         return res
 
-    def _add_positions_and_psf(self, medsifier, res, shear_str):
+    def _add_positions_and_psf(self, medsifier, res, shear_str, mbobs_list):
         """
         TODO add positionis etc.
         """
 
         new_dt = [
-            ('row', 'f4'),
-            ('col', 'f4'),
-            ('row_noshear', 'f4'),
-            ('col_noshear', 'f4'),
+            ('box_size', 'i4'),
+            ('row0', 'i4'),  # bbox row start
+            ('col0', 'i4'),  # bbox col start
+            ('row', 'f4'),  # row in image. Use row0 to get to global pixel coords
+            ('col', 'f4'),  # col in image. Use col0 to get to global pixel coords
+            ('row_noshear', 'f4'),  # noshear row
+            ('col_noshear', 'f4'),  # noshear col
             ('ormask', 'i4'),
+            ('mfrac', 'f4'),
             ('star_frac', 'f4'),  # these are for the whole image, redundant
```

Comment:
There are some old columns below here that are not useful anymore. We should remove those.